hydradx/model/model_initialization.py

```diff
@@ -1,49 +1,50 @@
 import pandas as pd
 
-def initialize_model(initial_liquidity, initial_tradevolume, initial_fee_assets, initial_fee_HDX):
 
-########## AGENT CONFIGURATION ##########
-# key -> token name, value -> token amount owned by agent
-# note that token name of 'omniABC' is used for omnipool LP shares of token 'ABC'
-# omniHDXABC is HDX shares dedicated to pool of token ABC
+def initialize_model(initial_liquidity, initial_tradevolume, initial_fee_assets, initial_fee_LRNA):
+
+    ########## AGENT CONFIGURATION ##########
+    # key -> token name, value -> token amount owned by agent
+    # note that token name of 'omniABC' is used for omnipool LP shares of token 'ABC'
+    # omniHDXABC is HDX shares dedicated to pool of token ABC
 
     trader = {'HDX': 1000000, 'R1': 1000000, 'R2': 1000000}
```

Comment:
Trader should have LRNA, not HDX?

Reply:
Yeah I was actually not sure about that one, since it would also be plausible for them to have HDX, and the tests passed...